README.md

```diff
@@ -96,43 +96,41 @@ For a list of all available commands run `npm run`.
 These commands are
 
 * building
-  * `npm run build` creates symlinks among packages and creates node, umd, and esm outputs for each package
+  * `npm run build` Creates esm and cjs outputs for each package
+  * `npm run build:esm` Creates esm outputs for each package
+  * `npm run build:cjs` Creates cjs outputs for each package
   * `npm run clean` runs `clean:src` and `clean:dist` _(requires bash console)_
     * `npm run clean:src` deletes `.d.ts`, `.js`, and `.js.map` files
     * `npm run clean:dist` deletes `.rpt2_cache` and `dist` folders
+  * `npm run deepclean` deletes node_modules and runs `clean`
   * `npm run lint` lints the TypeScript files
-  * `npm run lint:fix` lints the TypeScript files and fixes
+  * `npm run lint:fix` lints the TypeScript files and fixes what it can
   * `npm run prettify` beautifies TypeScript files
+  * `npm run getupdates` modifies package.json with npm updates to packages
 
 * testing
   * `npm run test` lints, then runs `test:chrome` tests to confirm that the API is functioning as expected
-  * `npm run test:browsers` runs karma in the Chrome, Firefox, and Chromium Edge browsers
```

Comment:
`npm run test:browsers` runs karma in the Chrome, Firefox, and Chromium Edge browsers. Learn more about Karma here  https://karma-runner.github.io/6.4/config/browsers.html